KosherCocoaTests/KCJewishCalendarTests.swift

```diff
@@ -29,6 +29,32 @@ class KCJewishCalendarTests: XCTestCase {
         super.tearDown()
     }
     
+    func testHolidays() {
+        let jewishCalendar = JewishCalendar()
+        
+        var lagComponents = DateComponents()
+        lagComponents.year = 5778
+        lagComponents.month = Int(HebrewMonth.iyar.rawValue)
+        lagComponents.day = 17//day is off because of adjustment with sunset, should be 18
```

Comment:
- Great callout! 
- Ideally, we would figure out this sunset offset thing. Not a blocker, don't worry about it for now.
- nit: Can we add a space before the `//`